commands/stonk.ts

```diff
@@ -0,0 +1,34 @@
+import { Message } from "discord.js";
+import { Command, CommandDefinition } from ".";
+import dotenv from "dotenv";
+
+dotenv.config();
+const fetch = require("node-fetch"); // Requiring fetch to make the stock API calls
+const FMP_KEY = process.env.FMP_API_KEY; // This will need to be added in the .env file
```

Comment:
If there's a way around having this here by all means let's do it

Reply:
This is a fine way to get the API key, just make sure you check if it's blank before you send a request, and raise an error that Heroku can see. `throw new Error("reason");`

Let's use an **ES6** import (like those above this) instead of a **CommonJS** import.

Reply:
sounds good!

Reply:
Actually @aMahanna new devs won't have the key, so make sure it handles no-key gracefully.

A new developer shouldn't have to grab API keys for every command.